src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -110,4 +110,8 @@ private ExportFormat format(Object format) {
     public boolean streamStatements() {
         return streamStatements;
     }
+
+    public boolean writeNodeProperties() {
```

Comment:
we should probably extract this once into field,
as it's used in the hot loop of the json generator